docs/book/src/developer/tilt.md

```diff
@@ -82,6 +82,52 @@ base64 -i ~/path/to/gcp/credentials.json
 }
 ```
 
+{{#/tab }}
+{{#tab AZURE}}
+
+You would prepare an Azure service principal. Here is an [azure-cli example](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest):
```

Comment:
let's make these instructions consistent with https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/getting-started.md#setting-up-your-azure-environment

Reply:
Thank you. I'll make these instructions consistent with that.